docs/05-technical-reference/03-architecture/obsv-03-architecture-tracing.md

```diff
@@ -2,8 +2,7 @@
 title: Tracing Architecture
 ---
 
-
```

Comment:
Collect traces flow: 
Merge point 2 and 3, summarizing that it eithers goes through port-forwarding directly to Jaeger or via ingress passing the oauth2-proxy, which assures authenticated users via configured OIDC provider

Reply:
Suggestion:

> The process of querying traces from Jaeger looks as follows:
> 
> 1. A Kyma user accesses the Jaeger UI to look for specific traces.
> 2. The user has [two options](../../../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana.md) to pass the request to the Jaeger Query service and retrieve the trace information: 
>   - If port forwarding has been set up, the user can access the Jaeger UI directly.
>   - If authentication using a OIDC provider has been configured, the request goes through the [Istio Ingress Gateway](TO_DO), which forwards the incoming connections to the service. Jaeger Query passes the request to the [Keycloak Gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) for authorization. The Gatekeeper calls the identity provider to authenticate the user and the request, and grants further access if the authentication is successful.
> 3. The Jaeger Deployment provides trace information.